### ISSUE TITLE
[BUG] state_template not hiding entities

### ISSUE THREAD

**Issue by Vogete** [joBr99/nspanel-lovelace-ui#655]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

According to <https://docs.nspanel.pky.eu/entities/#hide-item-based-on-ha-template> items can be hidden using `state_template`. This does not work, no matter how many different and simple/complex scenarios I try. 

For example, the template `{{is_state("input_boolean.test_boolean", "off")}}` returns the correct true/false value in the HA developer tools, but when I add `state_template: '{{is_state("input_boolean.test_boolean", "off")}}'` to the entity in `apps.yaml`, that never actually hides the entity from the NSPanel.
Using `state` or `state_not` works as expected (when I use `state: "on"` or similar).

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [X] Provide your apps.yaml (or your config):
```yaml
nspanel-1:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/tasmota_0CB430/RESULT"
    panelSendTopic: "cmnd/tasmota_0CB430/CustomSend"
    model: eu

    defaultCard: quickbuttons
    cards:
      - type: cardEntities
        title: QuickButtons
        entities:
          - entity: light.test_lights
            name: Test Lights
            icon: mdi:lightbulb-group
            state_template: '{{is_state("input_boolean.test_boolean", "off")}}'
```
- [X] Go to Settings>Add-ons>AppDaemon>Log (or the output of your TS script) and then, provide the output of the log during your issue/bug occurs:
```
Not relevant (I think. At least I couldn't see anything that was somewhat relevant to this. Please correct me if I'm wrong.)
```

### TO REPRODUCE
_Steps to reproduce the behavior:_

Create an input_boolean `test_boolean` in HA, add a light entity in NSPanel with `state_template: '{{is_state("input_boolean.test_boolean", "off")}}'`. 
Turn the input boolean on/off, and watch the entity in NSPanel lovelace not do anything, no matter the value of the boolean.

In the meantime, add `{{is_state("input_boolean.test_boolean", "off")}}'` into the template debug field in HA, and watch as it evaluates to the correct `true/false` values when you change the boolean.

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

When I add `state_template: '{{is_state("input_boolean.test_boolean", "on")}}'` to an entity on the NSPanel lovelace, I would expect the entity to disappear when the mentioned boolean is switched to True, then appear again when the boolean if False

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
_Please note here in case you are using ioBroker_

### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_

NSPanel EU version, 12.3.1 tasmota.
NSPanel Lovelace UI Backend v3.7.3

**Comment by Vogete** [comment]
hmm, this might be an issue with AppDaemon actually.

**Comment by joBr99** [comment]
hey, 
sorry for the confusion; this is a new feature that will be released with 3.8

https://github.com/joBr99/nspanel-lovelace-ui/issues/652

Please update to the development version to use it

https://docs.nspanel.pky.eu/faq/#how-to-upgrade-from-a-release-to-the-current-development-version

**Comment by Vogete** [comment]
ohh, I see. that makes more sense. I'll try that out! Thanks! :)
